.github/workflows/premerge.yml

```
@@ -0,0 +1,31 @@
+name: Premerge
+
+on:
+  pull_request:
+    types: [opened, synchronized, reopened]
```

```suggestion
  pull_request:
    types: [opened, synchronized, reopened]
  push:
    branches: [ main ]
  schedule:
    - cron: * * 1 * *
```

(doc: https://jasonet.co/posts/scheduled-actions/ )